client/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "openmsupply-client",
   "//": "Version is shared between client and remote server and is located in repo root package.json",
-  "version": "0.1.0",
+  "version": "1.1.1",
```

Comment:
Ravi - @andreievg changed this in a8e1c9f47e876843ba6e188e90762b33a73ae06e so that we could take the version from `package.json` up a level, as per the comment above. The idea is that we can share the version between server and client.